app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const app = express();
+let buzz = require('./routes/buzzwords')
+const buzzwords = require('./buzzwordStorage');
+
+app.use(express.static('public'));
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.use('/buzzwords', buzz);
+
+app.post('/reset', function (req, res) {
+  if (req.body.reset === true) {
+    buzzwords.reset();
+    res.json({ "success": true })
+  } else {
+    res.send(false)
+  }
+})
+
+app.post('/heard', function (req, res) {
+  if (req.body.buzzWord) {
+    res.json(buzzwords.score(req.body));
+  } else {
+    res.send(false)
+  }
+})
+
+const server = app.listen(4500, function () {
```

Comment:
Please declare a `PORT` variable at the top of the file and assign its value to `process.env.PORT || 4500`.